tests/hashed-apikey-rls.test.ts

```
@@ -10,7 +10,7 @@
  */
 import { randomUUID } from 'node:crypto'
 import { createClient } from '@supabase/supabase-js'
-import { Pool } from 'pg'
+import { Pool, type PoolClient } from 'pg'
```

ESLint

[error] Prefer using a top-level type-only import instead of inline type specifiers.

(import/consistent-type-specifier-style)